publish.gradle

```diff
@@ -251,7 +251,6 @@ ext.dependsOnTheMegazord = {
     // has their own build dir)
     android {
         sourceSets {
-            test.resources.srcDirs += "$buildDir/rustJniLibs/desktop"
```

Comment:
This line wasn't doing anything, because none of the component packages actually build any rust code into their own `$buildDir`; it's all in the megazord one below.